vignettes/quickstart.Rmd

```diff
@@ -49,7 +49,7 @@ data("antidepressant_data")
 dat <- antidepressant_data
 ```
 
-We consider an imputation model with the mean change from baseline in the HAMD17 score as the outcome (variable `CHANGE` in the dataset). The following covariates are included in the imputation model: the treatment group (`THERAPY`), the (categorical) visit (`VISIT`), treatment-by-visit interactions, the baseline HAMD17 score (`BASVAL`), and baseline HAMD17 score-by-visit interactions. A common unstructured covariance matrix structure is assumed for both groups. The analysis model is an ANCOVA model with the treatment group as the primary covariate and adjustment for the baseline HAMD17 score.
+We consider an imputation model with the mean change from baseline in the HAMD17 score as the outcome (variable `CHANGE` in the dataset). The following covariates are included in the imputation model: the treatment group (`THERAPY`), the (categorical) visit (`VISIT`), treatment-by-visit interactions, the baseline HAMD17 score (`BASVAL`), and baseline HAMD17 score-by-visit interactions. A common unstructured covariance matrix structure is assumed for both groups. The analysis model is an ANCOVA model with the treatment group as the primary factor and adjustment for the baseline HAMD17 score.
```

Comment:
Not sure if I like this,  as in yes technically from an implementation point of view it is stored as a factor but thats not what we are trying to draw attention to here is it? As in isn't the point of this sentence trying to say that treatment group is the main variable of interest, I.e. the variable we are drawing inference on?

Reply:
I think that calling the treatment group the "primary factor" in an ANCOVA model is in line with the wording of the theory of standard ANOVA (e.g. https://en.wikipedia.org/wiki/Analysis_of_variance#For_a_single_factor). The previous wording wasn't unclear to the JOSS reviewers.